hs_core/authentication.py

```diff
@@ -92,6 +92,4 @@ def authenticate(self, request):
             KEYCLOAK.token(decoded_username, decoded_password)
         except Exception:
             return None
-
-        user = User.objects.get(username=decoded_username)
-        return (user, None)
```

Comment:
not sure if there was a reason we were returning a tuple here...
https://docs.djangoproject.com/en/3.2/topics/auth/customizing/#writing-an-authentication-backend

Reply:
The example I was looking at returned the tuple, it must've been an older example.  No matter, it is not needed in our current version.